frontend/src/actions/TroopsActions.js

```diff
@@ -46,3 +46,25 @@ export const addTroopAction = kingdomID => {
     );
   };
 };
+
+export const upgradeTroopAction = (kingdomID, amount, level) => {
+  return dispatch => {
+    dispatch({
+      type: UPDATE_TROOPS,
+    });
+
+    return fetchKingdom
+      .put(kingdomID, 'troops', { amount: amount, level: level })
```

Comment:
@kkeszthe no need to specify keys if its the same as the name of var
```suggestion
      .put(kingdomID, 'troops', { amount, level })
```